telegraf.conf

```diff
@@ -1,138 +0,0 @@
-# Telegraf configuration
```

Comment:
Why was this removed? 👀

Reply:
Because it's not needed.

Reply:
@SoftCreatR I'm failing to understand why it is not needed. Could you please explain?

Reply:
The default config fits well in this case.